### ISSUE TITLE
Accessibility issues: missing alt text for images & screen reader issues in stash block

### ISSUE THREAD

**Issue by Nukker1** [abgreeve/moodle-block_stash#27]
There is currently no option to assign alternative texts for items in Stash, so they cannot be recognized when using a screen reader, which limits accessibility.
In addition, although the names of the items are displayed in the inventory, they cannot be read by the screen reader either. This means that a screen reader can neither read out the alternative texts of the items nor their names in the inventory.
